docs/man-pages/htcondor.rst

```diff
@@ -3,84 +3,169 @@
+the *noun* representing an object in the HTCondor system to be operated on.
+The nouns include an individual *job*, *jobset*, *eventlog*, or a *dag*.  Each
+noun is then followed by a noun-specific verb that describe the operation on
+that noun.
 
-For **jobs**, the **--resource** option allows you to run jobs on resources other than your
-local HTCondor pool. By specifying either ``EC2`` or ``Slurm`` here, the tool
-provisions resources for the time described by the **--runtime** option (in seconds) 
-and sends your HTCondor job to run on them. It assumes you have already 
-completed the necessary setup tasks, such as creating an account for Slurm
-submissions or making your AWS access keys available for EC2 submissions. 
-submissions.
+One of the following optional global option may appear before the noun:
 
 Global Options
 --------------
- **-h**, **--help**
+
+ **htcondor -h**, **htcondor --help**
      Display the help message. Can also be specified after any
-     subcommand to display the options available for each subcommand.
- **-q**
+     verb to display the options available for each verb.
+ **htcondor -q ...**
      Reduce verbosity of log messages.
- **-v**
+ **htcondor -v ...**
      Increase verbosity of log messages.
 
-Job Options
------------
-
- **--resource**
-    Resource type used to run this job. Currently supports ``Slurm`` and ``EC2``.
-    Assumes the necessary setup is complete and security tokens available.
- **--runtime**
-    Amount of time in seconds to allocate resources.
-    Used in conjunction with the *--resource* flag.
- **--email**
-    Email address to receive notification messages.
-    Used in conjunction with the *--resource* flag.
- **--skip-history**
-    Passed to the *status* subcommand to skip checking history
-    if job not found in the active job queue.
 
-Job Set Options
----------------
+Job Verbs
+---------
+
+ **htcondor job submit submit_file**
+     Takes as an argument a submit file in the *condor_submit* job submit
+     description language, and places a new job in an Access Point
+
+     **htcondor job submit Options**
+     
+          **htcondor job submit --resource resource_type submit_file**
+            Resource type used to run this job. Currently supports ``Slurm`` and ``EC2``.
+            Assumes the necessary setup is complete and security tokens available.
+          **htcondor job submit --runtime runtime_in_seconds submit_file**
+            Amount of time in seconds to allocate resources.
+            Used in conjunction with the *--resource* flag.
+          **htcondor job submit --email address submit_file**
+            Email address to receive notification messages.
+            Used in conjunction with the *--resource* flag.
+    
+    
+ **htcondor job status**
+     Takes as an argument a job id in the form of clusterid.procid,
+     and returns a human readable presentation of the status
+     of that job.
+
+     **job status option**
```

Comment:
See previous suggestions.  Be consistent.  Don't use the lower-case "option" because it makes it look even more like a command-line constant.